src/components/layout/breadcrumb/breadcrumb.tsx

```diff
@@ -8,7 +8,6 @@ const BreadcrumbContainer = styled.ol`
   list-style: none;
   display: flex;
   flex-wrap: nowrap;
-  overflow: hidden;
```

Comment:
We need this to prevent an overflow of the `BreadcrumbContainer`, e.g. on this page on mobile: https://deploy-preview-805--satellytes.netlify.app/blog/post/backstage-the-developer-portal-framework-for-optimizing-your-development-processes/

![Bildschirmfoto 2024-09-19 um 12 53 14](https://github.com/user-attachments/assets/1ef604dd-b0ad-48f0-a381-97f9e1d7fea7)